tests/unit/src/test/scala/tests/SbtSuite.scala

```diff
@@ -0,0 +1,82 @@
+package tests
+
+class SbtSuite extends BaseLspSuite("sbt") with ScriptsAssertions {
```

Comment:
Maybe we can just add it to `SbtLspSuite` in slow tests?

Reply:
Moved it into `SbtLspSuite`